### ISSUE TITLE
Metadata: Enable values with multiple lines

### ISSUE THREAD

**Issue by oz123** [trentm/python-markdown2#223]
Currenty, one can only use metadata the following way:

```
key: value
key2: value2
key3: value3
```

This is fine for simple case. However for my static site generator, I would like to have a long summary, which is easier to handle if spread over multiple lines. YAML has support for many ways to declare such values:
```
key: value
key2: |
     long value
     over mutliple lines
key3: >
      or this for example
      is another option
```

I have implemented a solution that monkey patched the class Markdown, and enables the second possibility. I think this would be useful for other people two. You can either still the code directly, or ask for a PR. 
I am sure it's still rough a bit (the REGEX might be better), but we can start from here.

You can find the relevant code here:

https://github.com/oz123/blogit/commit/6b73ee359b856c0a492256812607f41fa1c36eac

I simply override the method `_extract_metadata`.

**Comment by nicholasserra** [comment]
Hello, thanks for the info. I think the best route would be a PR that implements this functionality, but is also backwards compatible. Some added tests for the new cases would also be needed. Looks useful :)

**Comment by oz123** [comment]
PR is there, hope you like it!

**Comment by nicholasserra** [comment]
Fixed in #224
